feature/settings/src/main/res/values/strings.xml

```diff
@@ -83,5 +83,24 @@
     <string name="stabilization_description_on">Stabilization On</string>
     <string name="stabilization_description_high_quality">Stabilization High Quality</string>
     <string name="stabilization_description_off">Stabilization Off</string>
-    <string name="stabilization_description_unsupported">Stabilization unsupported</string>
+    <string name="stabilization_description_unsupported_device">Stabilization unsupported by device</string>
+    <string name="stabilization_description_unsupported_fps">Stabilization unsupported due to frame rate</string>
+
+
+    <!-- Target Fps setting strings -->
+    <string name="fps_title">Set Target Frame Rate</string>
+
+    <string name="fps_description_unavailable">No fixed frame rates supported</string>
+    <string name="fps_description_none">No target fps</string>
```

Comment:
When we drop "Target", we should just call this "Auto Frame Rate".

Reply:
let me know if the update is how you imagined the changes